crds/templates/endpoints.yaml

```diff
@@ -0,0 +1,63 @@
+{{- if or .Values.global.remotePilotCreateSvcEndpoint .Values.global.createRemoteSvcEndpoints }}
```

Comment:
seems like this shouldn't be in crd, as its not a crd.

Should we have a new "remote chart" that just has this remote stuff? I am not exactly sure how this is intended to be used so not sure what makes the most sense.

Reply:
This is going to have a ripple effect for operator too - we'd have to create a remote feature and component and stuff it into the API.  That seems like a bit change at the 11th hour.
Alternative: we leave it in here for now and decide how best to handle it in 1.5. It would mean a fairly small change to the docs if we do create a new chart for it.

Reply:
Per offline conversation, renaming to base, which is consistent with operator.